src/site/content/en/blog/codelab-sign-up-form-best-practices/index.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: codelab
+title: Sign-up form best practices codelab
+authors:
+  - samdutton
+scheduled: true
+date: 2020-11-06
+updated: 2020-11-06
+description: Use cross-platform browser features to build a simple sign-up form that's secure, accessible and easy to use.
+tags:
+  - forms
+  - identity
+  - layout
+  - mobile
+  - security
+  - ux
+glitch: sign-up-form-codelab-0
```

Comment:
@samdutton This glitch is not working and I can't find anything under that name on your glitch profile. There's `singup-form` and `singup-form-codelab` though.